src/transformers/models/splinter/tokenization_splinter_fast.py

```diff
@@ -0,0 +1,136 @@
+# coding=utf-8
+# Copyright Ori Ram, Yuval Kirstain, Jonathan Berant, Amir Globerson, Omer Levy and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tokenization classes for Splinter."""
+from typing import List, Optional
+
+from ...utils import logging
+from ..bert.tokenization_bert_fast import BertTokenizerFast
+from .tokenization_splinter import SplinterTokenizer
+
+
+logger = logging.get_logger(__name__)
+
+VOCAB_FILES_NAMES = {"vocab_file": "vocab.txt"}
+
+PRETRAINED_VOCAB_FILES_MAP = {
+    "vocab_file": {
+        "splinter-base": "https://huggingface.co/splinter-base/resolve/main/vocab.txt",
+        "splinter-large": "https://huggingface.co/splinter-large/resolve/main/vocab.txt",
+    }
+}
+
+PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES = {
+    "splinter-base": 512,
+    "splinter-large": 512,
+}
+
+PRETRAINED_INIT_CONFIGURATION = {
+    "splinter-base": {"do_lower_case": False},
+    "splinter-large": {"do_lower_case": False},
+}
+
+
+class SplinterTokenizerFast(BertTokenizerFast):
+    r"""
+    Construct a fast Splinter tokenizer.
+
+    :class:`~transformers.SplinterTokenizer` is identical to :class:`~transformers.BertTokenizer` and runs end-to-end
```

Comment:
```suggestion
    :class:`~transformers.SplinterTokenizerFast` is identical to :class:`~transformers.BertTokenizerFast` and runs end-to-end
```

Plus same comment as above: the difference with the BERT tokenizer should be highlighted.